### PR TITLE
general content changes

### PR DESCRIPTION
**Pull request by agravell047**
## Summary

Made general content and text changes across MHV apps and updated unit tests to reflect those changes.

## Related issue(s)

https://jira.devops.va.gov/browse/MHV-50830

> -Breadcrumb shows as "About Medications" needs to show as "About medications" with a lower case m
-"How to request a renewal" accordion on landing page has a missing space between the SM link and next line of text:
-Non-VA details page is showing the wrong H2 "About your prescription." It should say "About this medication or supply."
-Medications that have never been filled are showing "First fill" with empty dates in Refill history section. We should change the tool to remove the Refill history section entirely when a prescription has never been filled. 
-"No Image Available" on details page should be "No image available" with lower case i and a. 
-The "P" in Refill in process status should not be capitalized
-Learn how to manage you VA prescriptions text font and sizing is incorrect
-Every time it says Blue button must have registration mark
-Above screenshot should also have a period between website and Go (website. Go to VA Blue Button...)
-"What to know before you download" should be "What to know before you print or download"
-'Download your medication details as a PDF' should change to 'Download this page as a PDF'

## Testing done

- Tested locally
- Ran unit tests locally 

Updated unit and cypress tests, new ones weren't needed because of small content changes.

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Mobile  |        |       |
| Desktop |  Screenshot 2023-11-15 at 10 09 07 AM |  Screenshot 2023-11-15 at 9 50 05 AM |
| Desktop |  Screenshot 2023-11-15 at 10 08 54 AM | Screenshot 2023-11-15 at 9 51 43 AM |
 Desktop | Screenshot 2023-11-15 at 10 04 39 AM | Screenshot 2023-11-15 at 9 51 49 AM |
  Desktop | Screenshot 2023-11-15 at 10 08 43 AM | Screenshot 2023-11-15 at 10 01 20 AM|

## What areas of the site does it impact?

MHV medications landing page, prescriptions list page and details page. Also included text change to medical records page as mentioned in the ticket.

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [] Browser console contains no warnings or errors.
- [] Events are being sent to the appropriate logging solution
- [] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Review by powellkerry — APPROVED** [review]

**Review by mmoyer-va — APPROVED** [review]

**Review by acrollet — COMMENTED** [review]
I'm curious why this is not using the HTML entity and the others are?

File: src/applications/mhv/medications/components/MedicationsList/MedicationsListCard.jsx [hunk 1]
```diff
@@ -45,7 +45,9 @@ const MedicationsListCard = props => {
             id="status"
             className="vads-u-margin-top--1p5 vads-u-font-weight--bold"
           >
-            {rx.dispStatus}
+            {rx.dispStatus !== 'Active: Refill in Process'
```
Comment (acrollet) [inline src/applications/mhv/medications/components/MedicationsList/MedicationsListCard.jsx:48]: Should we be using a constant here?

File: src/applications/mhv/medications/components/shared/ExtraDetails.jsx [hunk 2]
```diff
@@ -50,6 +50,13 @@ const ExtraDetails = rx => {
           updates.
         </p>
       )}
+      {dispStatus === dispStatusObj.activeParked && (
```
Comment (acrollet) [inline src/applications/mhv/medications/components/shared/ExtraDetails.jsx:53]: Ideally, I think I would like to see a helper method With a clear name used in cases like these. (Maybe something like 
`prescriptionIsParked`)

**Review by acrollet — APPROVED** [review]
I'm leaving on approval because I'm traveling tomorrow and don't want to be a blocker. However, I would really like to see any logic that has to do with determining the status of a prescription (like parked) to use a helper method so that we can ensure we are always using the same logic when we don't get a status directly from the back end. Thanks for the cleanup work!

**Review by radavis — APPROVED** [review]

**Comment by agravell047** [comment]
> I'm curious why this is not using the HTML entity and the others are?

@acrollet Sorry, I'm not sure what you're referencing with this comment. Can you give a little more context?

**Comment by agravell047** [comment]
> I'm leaving on approval because I'm traveling tomorrow and don't want to be a blocker. However, I would really like to see any logic that has to do with determining the status of a prescription (like parked) to use a helper method so that we can ensure we are always using the same logic when we don't get a status directly from the back end. Thanks for the cleanup work!

@acrollet Okay, thank you! I would be happy to clean this up in another ticket or PR. My worry is that because these constants are used all throughout the code, it would make this a much bigger PR and I would want to take the time to make sure nothing breaks.

File: src/applications/mhv/medications/components/MedicationsList/MedicationsListCard.jsx (hunk 1 above)
Comment (agravell047) [reply src/applications/mhv/medications/components/MedicationsList/MedicationsListCard.jsx:48]: I agree that would be a good idea. I would want to replace other places where this logic is used as well if so.

**Pull request merged**